src/tiles/list/tile-list-compact.ts

```diff
@@ -1,20 +1,104 @@
 import { css, html, LitElement } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { SortParam } from '@internetarchive/search-service/dist/src/search-params';
 import { TileModel } from '../../models';
+import { formatCount, NumberFormat } from '../../utils/format-count';
+import { formatDate, DateFormat } from '../../utils/format-date';
+import '../../mediatype-icon';
+
+/*
+at 750 creator, title trimmed
+at 530
+
+css elipsis property
+*/
 
 @customElement('tile-list-compact')
 export class TileListCompact extends LitElement {
   @property({ type: Object }) model?: TileModel;
 
+  @property({ type: String }) baseNavigationUrl?: string;
+
+  @property({ type: Number }) currentWidth?: number;
+
+  @property({ type: Number }) currentHeight?: number;
+
+  @property({ type: Object }) sortParam?: SortParam;
+
   render() {
+    // Todo: Manage different date types
     return html`
-      <h1>${this.model?.title}</h1>
-      <h2>${this.model?.datePublished?.toDateString()}</h2>
+      <div id="list-compact" class="${this.classSize}">
+        <div id="views">
+          ${formatCount(this.model?.viewCount ?? 0, this.formatSize)}
+        </div>
+        <div id="title">
+          <a
+            href="${this.baseNavigationUrl}/details/${this.model?.identifier}"
+            title=${ifDefined(this.model?.title)}
+          >
+            ${this.model?.title}
+          </a>
+        </div>
+        <div id="date-published">${formatDate(this.date, this.formatSize)}</div>
+        <div id="creator">${this.model?.creator}</div>
+        <div id="icon">
+          <mediatype-icon
+            mediatype="${ifDefined(this.model?.mediatype)}"
```

Comment:
When you're binding properties in Lit, you typically want to prefix the property name with a dot `.` so it uses Lit's binding which is much more powerful than html binding. ie:
```
.mediatype=${this.model?.mediatype}
```

Reply:
@jbuckner Let's talk about this. Getting an odd message. May need to change `mediatype-icon` slightly to handle `undefined`

Reply:
Changed mediatype-icon